universe/vikram.hello/index.js

```diff
@@ -0,0 +1,194 @@
+                    const castQuery = "https://api.themoviedb.org/3/movie/" + result.id + "/credits?api_key=654083fa9e049ea234e4ae94d3e65774&language=en-US";
+                    let oneDate = new Date(Date.now());
+                    if (String(result.release_date) != ''){
+                        oneDate = new Date(result.release_date);
+                    }
+                    const movieObj = {
+                        id: new Tp.Value.Entity(String(result.id), String(result.title)),
+                        description: result.overview,
+                        release_date: oneDate,
+                        rating_score: Number(result.vote_average),
+                        actors:[]
+                    }
+                    try{
+                        const actorResponse = await Tp.Helpers.Http.get(castQuery);
+                        const actorsParsed = JSON.parse(actorResponse);
+                        for (const person1 of (actorsParsed.cast)){
+                            movieObj.actors.push(new Tp.Value.Entity(String(person1.id), person1.name));
+                        }
+                    }
+                    catch(err){
+                        var newArr = [];
+                        movieObj.actors.push(newArr);
+                    }
+                return movieObj;
+            }));
+            
+        }
+}
+    get_person(params, hints, env){
+        let actorSortUrl = '';
+        let actorQueryURL = "https://api.themoviedb.org/3/search/person?api_key=" + this.constructor.metadata.auth.api_key + "&language=en-US&query=";
+        if (hints && hints.sort){
+            if (hints.sort[0] === 'popularity' && hints.sort[1] === 'desc')
+                actorSortUrl = "https://api.themoviedb.org/3/person/popular?api_key=" + this.constructor.metadata.auth.api_key + "&language=en-US&page=1";
+        }
+        if (actorSortUrl){
+            return Tp.Helpers.Http.get(actorSortUrl).then((response) => {
+                let parsedResponse = JSON.parse(response);
+                return parsedResponse.results.map((result) => {
+                    let id = new Tp.Value.Entity(String(result.id), result.name);
+                    return ({
+                        id,
+                        popularity:result.popularity,
+                    });
+                });
+            });
+        }
+        let actorQuery = '';
+        if (hints && hints.filter){
+            for (let [pname, op, value] of hints.filter) {
+                if (pname === 'id' && (op === '==' || op === '=~')) {
+                    if (value instanceof Tp.Value.Entity)
+                        actorQuery = encodeURIComponent(value.display);
+                        console.log("Hi");
+                        console.log(actorQuery);
+                }
+            }
+        }
+        if (!actorQuery) {
+            console.log("Hi");
+            actorQuery = 'Brad%20Pitt';}
+            console.log(actorQuery);
```

Comment:
Indentation here is wrong and a bit funky. Please clean the indentation everywhere.